gems/sorbet/test/snapshot/run_one_bazel.sh

```diff
@@ -0,0 +1,176 @@
+    export RUNFILES_DIR="$0.runfiles"
+  fi
+fi
+if [[ -f "${RUNFILES_DIR:-/dev/null}/bazel_tools/tools/bash/runfiles/runfiles.bash" ]]; then
+  # shellcheck disable=SC1090
+  source "${RUNFILES_DIR}/bazel_tools/tools/bash/runfiles/runfiles.bash"
+elif [[ -f "${RUNFILES_MANIFEST_FILE:-/dev/null}" ]]; then
+  # shellcheck disable=SC1090
+  source "$(grep -m1 "^bazel_tools/tools/bash/runfiles/runfiles.bash " \
+            "$RUNFILES_MANIFEST_FILE" | cut -d ' ' -f 2-)"
+else
+  echo >&2 "ERROR: cannot find @bazel_tools//tools/bash/runfiles:runfiles.bash"
+  exit 1
+fi
+# --- end runfiles.bash initialization ---
+
+repo_root="$PWD"
+
+# NOTE: using rlocation here because this script gets run from a genrule
+# shellcheck disable=SC1090
+source "$(rlocation com_stripe_ruby_typer/gems/sorbet/test/snapshot/logging.sh)"
+
+# ----- Option parsing -----
+
+# these positional arguments are supplied in snapshot.bzl
+ruby_package=$1
+output_archive="${repo_root}/$2"
+test_name=$3
+
+if [[ "${test_name}" =~ partial/* ]]; then
+  is_partial=1
+else
+  is_partial=
+fi
+
+info "├─ test_name:  ${test_name}"
+info "├─ output_archive:  ${output_archive}"
+info "├─ is_partial: ${is_partial:-0}"
+
+# ----- Environment setup -----
+
+# Add ruby to the path
+# shellcheck disable=SC1090
+PATH="${repo_root}/$(dirname "$(rlocation "${ruby_package}/ruby")"):$PATH"
+export PATH
+
+# Add bundler to the path
+# shellcheck disable=SC1090
+BUNDLER_LOC="${repo_root}/$(dirname "$(rlocation "gems/bundler/bundle")")"
+GEMS_LOC="$BUNDLER_LOC/../gems"
+PATH="$BUNDLER_LOC:$PATH"
+export PATH
+
+test_dir="${repo_root}/gems/sorbet/test/snapshot/${test_name}"
+
+actual="${test_dir}/actual"
+
+srb="${repo_root}/gems/sorbet/bin/srb"
+
+# Use the sorbet executable built by bazel
+SRB_SORBET_EXE="$PWD/$(rlocation com_stripe_ruby_typer/main/sorbet)"
+export SRB_SORBET_EXE
+
+HOME=$test_dir
+export HOME
+
+XDG_CACHE_HOME="${test_dir}/cache"
+export XDG_CACHE_HOME
+
+info "├─ ruby: $(command -v ruby)"
+info "├─ ruby --version: $(ruby --version)"
+
+info "├─ bundle: $(command -v bundle)"
+info "├─ bundle --version: $(bundle --version)"
+
+info "├─ sorbet: $SRB_SORBET_EXE"
+info "├─ sorbet --version: $("$SRB_SORBET_EXE" --version)"
+
+
+# ----- Build the test sandbox -----
+
+cp -r "${test_dir}/src" "$actual"
+
+
+# ----- Run the test -----
+
+(
+  cd "$actual"
+
+  # Setup the vendor/cache directory to include all gems required for any test
+  info "├─ Setting up vendor/cache"
+  mkdir vendor
+  ln -s "$GEMS_LOC" "vendor/cache"
+
+  info "├─ Testing 'bundle exec which ruby'"
+  if bundle exec which ruby ; then
+    info "├─ success"
+  else 
+    error "└─ failure"
+    exit 1
+  fi
+
+  # Configuring output to vendor/bundle
+  # Passing --local to never consult rubygems.org
+  # Passing --no-prune to not delete unused gems in vendor/cache
+  info "├─ Installing dependencies to vendor/bundle"
+  bundle config --local path vendor/bundle
```

Comment:
How is path meaningful? If it's an actual file path, I'd prefer `./path`, if it's some sort of enum that `--local` accepts, I'd prefer `"path"`

Reply:
This installs to `vendor/bundle` instead of `.bundle`. It's not super meaningful, so I'll try testing without it.

Reply:
It wasn't necessary. Since this is running in the sandbox, it won't matter where the installed gems go.